backend/LexBoxApi/Models/Project/CreateProjectInput.cs

```diff
@@ -10,5 +10,6 @@ public record CreateProjectInput(
     [property: MinLength(4), RegularExpression(LexCore.Entities.Project.ProjectCodeRegex)]
     string Code,
     ProjectType Type,
-    RetentionPolicy RetentionPolicy
+    RetentionPolicy RetentionPolicy,
+    Guid? RequestingUserId
```

Comment:
from an API perspective this should probably be called `ProjectManagerId`, otherwise it looks like it would be the Id of the user executing the mutation to create the project.